packages/components/src/Chips/utils/mdxUtils.ts

```diff
@@ -0,0 +1,46 @@
+import { uniq } from "lodash";
+import { useEffect, useState } from "react";
+
+export function useFakeOptionQuery() {
+  const [options, setOptions] = useState<string[]>([]);
+  const initialDataGetUrl = "https://swapi.dev/api/people/?format=json";
```

Comment:
This is a funny idea 😛 I am wondering if we can just use an internal mock list to avoid the API call, or if this is something you wanted to mimic?

When I also hold down tab to select all of the characters that are present, it then takes a sec to reload, so I wonder if this call is related?

Reply:
Got that from the `useCollectionQuery` but opted with a simpler pure js approach since there's a lot of setup involved with using that hook. The real API call is intended to give a better feel for it.

> When I also hold down tab to select all of the characters that are present, it then takes a sec to reload, so I wonder if this call is related?

like hold and click? that won't work since it resets on every select

Reply:
> Fixed

Woot woot!

> Haven't really done research in to those keyboard button, but I believe most modern laptop keyboard doesn't even have those keys

I think you can maybe do the same thing with like a `fn + up arrow`

> Screenshot? Seems to work fine on the deploy preview

I tried again and it seems like if you scroll to the very bottom while loading it will show the whole thing, but cuts off half of it while it's loading?  Maybe it's a me problem, because that recording you took looks good!  Here's me interacting with it anyways:

https://user-images.githubusercontent.com/88052089/151081272-7973346d-bb5c-4bd3-9bf8-f5a91d0c0dc4.mov

Reply:
>like hold and click? that won't work since it resets on every select

I mean like just hold tab key down and let all of the selections on the menu be chosen really fast like.  Although, weirdly enough I did just try it again and it wasn't an issue!

Reply:
> Should we make a note that there is a call being made in the mdx file

Whatcha worried about?

> I mean like just hold tab key down and let all of the selections on the menu be chosen really fast like. Although, weirdly enough I did just try it again and it wasn't an issue!

Ah yeah, not worried about that. There's no native interaction where you hold a tab and it doing something expected. (Like holding the shift and pressing a letter capitalizes it.)

Reply:
> Whatcha worried about?

I was just thinking the data fetching might take a while for some reason, and someone will wonder "hmm, I wonder why these characters are taking a while to load"

Reply:
Haven't heard that issue from `useCollectionQuery` so I wouldn't worry about it too much